lib/gds_api/email_alert_api.rb

```diff
@@ -77,6 +77,15 @@ def unsubscribe(uuid)
     post_json("#{endpoint}/unsubscribe/#{uuid}")
   end
 
+  # Unsubscribe from everything
+  # #
+  # @param integer Subscriber id
+  #
+  # @return [Hash] deleted
+  def unsubscribe_all(id)
```

Comment:
I would call this `unsubscribe_subscriber` (and then I would call the other one `unsubscribe_subscription` but that's already in use :cry:), but I don't really mind.

Reply:
Done :)